help_to_heat/templates/frontdoor/feedback.html

```diff
@@ -2,7 +2,7 @@
 {% import "macros.html" as macros with context %}
 
 {% block title %}
-  <title>{{_("Give us some feedback - Check eligibility for Great British Insulation Scheme - GOV.UK")}}</title>
+  <title>{{_("Help us improve the service - Check eligibility for Great British Insulation Scheme - GOV.UK")}}</title>
```

Comment:
Changing as I thought slightly more delicate wording would be more appropriate 😅

Reply:
As long as the client is happy with this change :D

Reply:
What do you think, @CheshireSwift? I don't think there have been any specific requirements around titles, iirc?

Reply:
Hmm. I thought there was a standardised title format beta feedback pages, but it doesn't look like it. Still, it's user-facing content, so I'd want to check it with stakeholders, yeah.

Reply:
If we think the change is worth it I'll ask, otherwise I can just leave it as is? @CheshireSwift

Reply:
If they want it out ASAP I'd be tempted to just leave it as it was 🙂